README.md

```diff
@@ -145,6 +145,10 @@ To deploy your `allinone` environment via `Vagrant` simply run:
 $ bin/run_vagrant up allinone
 ```
 
+Note: The default OS for ursula is Ubuntu Precise, if you want Trusty set
```

Comment:
Sure,  that note only applies to `Vagrant` though ofc.   otherwise the default OS is the one you're deploying to. :)